pyband/transaction.py

```diff
@@ -1,37 +1,24 @@
-import base64
-import json
-from .client import Client
-from typing import List, Optional
-from .wallet import PublicKey
+from .wallet import PrivateKey
 from .constant import MAX_MEMO_CHARACTERS
-from .message import Msg
 from .exceptions import EmptyMsgError, NotFoundError, UndefinedError, ValueTooLargeError
+from google.protobuf import any_pb2
+from pyband.proto.oracle.v1 import tx_pb2 as tx_oracle_type
+from pyband.proto.cosmos.tx.v1beta1 import tx_pb2 as cosmos_tx_type
+from pyband.proto.cosmos.tx.signing.v1beta1 import signing_pb2 as tx_sign
 
 
 class Transaction:
     def __init__(self):
-        self.msgs: List[Msg] = []
+        self.msgs: List[any_pb2.Any] = []
         self.account_num: Optional[int] = None
         self.sequence: Optional[int] = None
         self.chain_id: Optional[str] = None
-        self.fee: int = 0
+        self.fee: cosmos_tx_type.Fee = None
         self.gas: int = 200000
         self.memo: str = ""
 
-    def with_messages(self, *msgs: Msg) -> "Transaction":
-        self.msgs.extend(msgs)
-        return self
-
-    def with_auto(self, client: Client) -> "Transaction":
```

Comment:
Why does this function got removed?

I think it would be great if we have this function to automatically assign account number and sender address based on given messages.